samples/DFRobotDFPlayerMini/readme.md

```diff
@@ -0,0 +1,4 @@
+Steps to test this sample:
```

Comment:
1. Please, put a line explaining what DFPlayerMini is. Something like "DFPlayer - A Mini MP3 Player. See ... for details.".
2. Rename the sample directory from `DFRobotDFPlayerMini` to the shorter `DFPlayerMini`. 
3. It would be great if you can add also a link to your YouTube video.

Reply:
Directory renamed. More explanation added to read-me file. I will create a bit more polished video later. I don't have a spare speaker :). The sample created for evidence used a headphone.